src/pal/src/loader/module.cpp

```diff
@@ -1512,7 +1524,7 @@ static MODSTRUCT *LOADAddModule(void *dl_handle, LPCSTR libraryNameOrPath)
             {
                 module->refcount++;
             }
-            dlclose(dl_handle);
+            // dlclose(dl_handle);
```

Comment:
Mistake?

Reply:
No, I added you to a thread about this.

Reply:
Okay. Thanks. Please don't simply comment out code, it breeds questions that are impossible to answer after development. Remove the code if it is not needed.